src/content/docs/zh-cn/features/stronghold.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Stronghold
+description: 加密的、安全的数据库。
+---
+
+import PluginLinks from '@components/PluginLinks.astro';
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+<PluginLinks plugin="stronghold" />
+
+使用 [IOTA Stronghold](https：//github.com/iotaledger/stronghold.rs) 加密数据库和安全运行时存储秘密和密钥。
+
+## 支持的平台
+
+- Windows
+- Linux
+- macOS
+
+## 设置
+
+_这个插件要求 Rust 版本至少是  **1.75**_
+
+安装 stronghold 插件开始。
+
+<Tabs>
+	<TabItem label="自动" >
+		使用项目的包管理器来添加依赖。
+
+    	{ ' ' }
+
+    	<CommandTabs
+    		npm="npm run tauri add stronghold"
+    		yarn="yarn run tauri add stronghold"
+    		pnpm="pnpm tauri add stronghold"
+    		cargo="cargo tauri add stronghold"
+    	/>
+    </TabItem>
+
+    <TabItem label = "手动">
+    	<Steps>
+
+    	1. 在你的 `Cargo.toml` 文件中添加以下内容来安装 stronghold 插件。
+
+    		```toml title="src-tauri/Cargo.toml"
+    		[dependencies]
+    		tauri-plugin-stronghold = "2.0.0-beta"
+    		# 或者使用 Git：
+    		tauri-plugin-stronghold = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+    		```
+
+    	2. 修改 `lib.rs` 来初始化插件。
+
+    		```rust title="src-tauri/src/lib.rs" ins={3}
+    		fn run() {
+    				tauri::Builder::default()
+    						.plugin(tauri_plugin_stronghold::Builder::new(|password| {}).build())
+    						.run(tauri::generate_context!())
+    						.expect("error while running tauri application");
+    		}
+    		```
+
+    	3. 使用你喜欢的 JavaScript 包管理器安装 JavaScript Guest 绑定。
+
+    		<CommandTabs
+    			npm = "npm install @tauri-apps/plugin-stronghold"
+    			yarn = "yarn add @tauri-apps/plugin-stronghold"
+    			pnpm = "pnpm add @tauri-apps/plugin-stronghold"
+    		/>
+
+    	</Steps>
+    </TabItem>
+
+</Tabs>
+
+## 用法
+
+### 使用自定义密码散列函数初始化
+
+```rust title="src-tauri/src/lib.rs"
+pub fn run() {
+		tauri::Builder::default()
+				.plugin(
+						tauri_plugin_stronghold::Builder::new(|password| {
+								// 在这里使用 argon2, blake2b 或任何其他安全算法对密码进行散列。
```

Comment:
```suggestion
								// 在这里使用 argon2、blake2b 或任何其他安全算法对密码进行散列。
```

Reply:
Might want to be careful committing this on GitHub as it tends to break the formatting.